app/main.py

```diff
@@ -1 +1,39 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self, name: str, appetite: int, is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, 3, is_hungry)
```

Comment:
```suggestion
      super().__init__(name=name, appetite=7, is_hungry=is_hungry)
```
change it in all of the code